### ISSUE TITLE
Ignore releases by special purpose artists

### ISSUE THREAD

**Issue by LeoVerto** [schnatterer/nusic#5]
Musicbrainz uses a number of [special purpose artists](https://musicbrainz.org/doc/Style/Unknown_and_untitled/Special_purpose_artist), the most common one being [Various Artists](https://musicbrainz.org/artist/89ad4ac3-39f7-470e-963a-56509c546377).

If a user has a single release tagged to, say "Various Artists", nusic will display hundreds of mostly unrelated new releases.
I know it's possible to ignore all releases by that artist but it could be quite confusing for new users.

Lastly, thanks for using MB data in such a great way and publishing your app on F-Droid! :smiley:

**Comment by schnatterer** [comment]
@LeoVerto Thanks for your constructive feedback! Actually, several users got confused by this behavior already. That's why I included the "Hide all releases" feature on long tabbing a release you referred. I do like your idea of simplifying this process for new users, and the link to the special purpose artists is a good default list to start with. I'll just have to figure out how to combine the approaches of "hidden" artists and ignored artists by keeping it as simple as possible.

**Comment by robin-paulson** [comment]
> "Various Artists", nusic will display hundreds of mostly unrelated new releases

Yes, I got this behaviour too this week. Frustrating. How likely are you to fix it? Thanks
